charts/honeycomb/README.md

```diff
@@ -80,7 +80,8 @@ The following table lists the configurable parameters of the Honeycomb chart, an
 |---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------|
 | `honeycomb.apiKey`              | Honeycomb API Key                                                                                                                                                                                                                           | `YOUR_API_KEY`                                    |
 | `honeycomb.apiHost`             | API URL to sent events to                                                                                                                                                                                                                   | `https://api.honeycomb.io`                        |
-| `honeycomb.existingSecret`      | Name of an existing secret resource to use containing your API Key in the `api-key` field                                                                                                                                                   | `nil`                                             |
+| `honeycomb.existingSecret`      | Name of an existing secret resource to use containing your Honeycomb API Key                                                                                                                                                                | `nil`                                             |
```

Comment:
There's also a reference to the `api-key` field in line 71, not sure if needs updating as well

Reply:
ah good catch, updating